Assets/XML/GameText/Bonus_CIV4GameText.xml

```diff
@@ -7800,7 +7800,7 @@
 		<Russian>Ðèìëÿíå</Russian>
 	</TEXT>
 	<TEXT>
-		<Tag>TXT_KEY_BONUS_ROMANI</Tag>
+		<Tag>TXT_KEY_BONUS_ROMANE</Tag>
 		<English>Romanì</English>
 		<Russian>Öûãàíå</Russian>
```

Comment:
Looks like you have a mistake here.

Reply:
Hey Toffer, please read the comment above it's intentional, but if you prefer I'll change them back.

Reply:
Please let me know if I should do something.